report_html_stock.py

```diff
@@ -69,3 +69,45 @@ def parse(cls, report, records, data, localcontext):
         return super(PickingList, cls).parse(
             report, records, data, localcontext
         )
+
+
+class SupplierRestockingList(ReportMixin):
+    'Supplier Restocking List'
+    __name__ = 'report.supplier_restocking_list'
+
+    @classmethod
+    def parse(cls, report, records, data, localcontext):
+
+        sorted_moves = {}
```

Comment:
I think this code can be written as a method in ReportMixin class which all reports can use